### PR TITLE
@cavvia => Replace /collect grid with <ArtworkGrid />

### PR DESCRIPTION
**Pull request by damassi**
First replacement location for new stitch work. Note that once this is in the wild and sufficiently tested we can start removing *a lot* of code.

#### Adds:
- `ENABLE_EXPERIMENTAL_STITCH_INJECTION` feature flag. To view new columns, update `.env` and set to `true`

#### Todo (different PR): 
- Some warning cleanup in the console

### CODE REVIEW

File: src/desktop/lib/global_react_modules.js [hunk 1]
```diff
@@ -1,153 +1,19 @@
...
-  props
+export const Artwork = props => (
+  <_Artwork artwork={artwork} {...props} useRelay={false} />
+)
+export const ArtworkGrid = props => (
+  <_ArtworkGrid artworks={artworks} {...props} useRelay={false} />
+)
+export const Fillwidth = props => (
+  <_Fillwidth artworks={artworks} {...props} useRelay={false} />
+)
```
Comment (damassi) [inline src/desktop/lib/global_react_modules.js]: This file will be cleaned up in a subsequent PR, using it as a bit of a testbed for now

File: src/desktop/components/artwork_columns/view.coffee [hunk 2]
```diff
@@ -74,15 +76,30 @@ module.exports = class ArtworkColumns extends Backbone.View
...
-    @$el
-      .html artworkColumns
-        numberOfColumns: @numberOfColumns
-        buttonLabel: @buttonLabel()
-        seeMore: @seeMore
 
-    @sizeColumns()
-
-    @appendArtworks @collection.models
+    if ENABLE_EXPERIMENTAL_STITCH_INJECTION
```
Comment (cavvia) [inline src/desktop/components/artwork_columns/view.coffee]: i'm concerned we'll create a proliferation of these feature flag switches in our code - is there a reason we don't want to just QA this and roll it out?

Comment (cavvia) [reply src/desktop/components/artwork_columns/view.coffee]: OK great - as discussed, we'll enable it on Staging for a period.

**Comment by damassi** [comment]
(Merging since its 🍏)

**Pull request merged**